package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "ng-rolodex",
   "version": "0.0.0",
+  "proxy": "http://localhost:8080",
```

Comment:
This is not the correct way to setup the proxy for angular: https://github.com/angular/angular-cli/blob/master/docs/documentation/stories/proxy.md